changes/issue-8506-remove-Preview-payload-button

```diff
@@ -0,0 +1,2 @@
+* Remove the Preview payload button from the usage statistics page, as well as its associated logic
```

Comment:
The change log is turned into user facing documentation in the release, so no code detail needed here. ~~as well as its associated logic and unique styles~~

You can add that detail on the PR itself <3

Reply:
Ah, I see, yes that makes sense. Thanks.